src/hooks/common.ts

```diff
@@ -1,9 +1,10 @@
 import { memoizeFunction } from "@fluentui/react";
 import axios from "axios";
+import { getTokenFromLocalStorage } from "@/utils/auth";
 
 export const getBaseUrl = () => {
   // return _getLocalUrl();
-  return "https://api.itx-app.com";
+  return "https://api.ppe.itx-app.com";
```

Comment:
largest ask from review - can we make this env specific? maybe a part of the launch build on vite/bun's part or something in webpack or somethin - never dealt with env vars in frontend so I trust you know better. 

we need this since the ppe/prod frontend versions are gonna essentially point to different backends as well.

Reply:
current solution is use .env with a var called VITE_ITX_API_URL